### PR TITLE
Contact form

### PR DESCRIPTION
**Pull request by kkmanuu**
**Added Features**
Added index.html contact form.
Added style.css

### CODE REVIEW

**Review by skyv26 — COMMENTED** [review]
## Hi @kkmanuu, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request 🟡 

You have submitted a project which doesn't have a coding partner contribution. Kindly make sure that `pair-programming` needs contributions from both parties and please ask your coding partner to contribute in your project.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by kkmanuu** [comment]
Enhancing collaboration by inviting my esteemed coding partner to contribute and make valuable additions to the project.

File: index.html [hunk 1]
```diff
@@ -292,5 +292,47 @@ <h3 class="skillset-heading">Skills</h3>
...
+        required
+      />
+      <textarea
+        class="contact-message"
+        name="user-message"
+        maxlength="500"
+        placeholder="Write the message here"
+        required
+      ></textarea>
+      <button class="contact-btn" type="submit">Get in touch</button>
```
Comment (divinecharlotte) [inline index.html:333]: - [ ] Kindly consider displaying this button on the left side not in the center as you did to implement exactly the provided design check the image below for more clarification
- your design 👇 
Screen Shot 2023-05-10 at 10 58 47


- Figma design 👇 
Screen Shot 2023-05-10 at 11 00 13

**Review by divinecharlotte — CHANGES_REQUESTED** [review]
Hi @Team   , 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

## To Highlight 💯 
- descriptive pr👍
- linters are passing 👍 




 

## Required Changes ♻️

_Check the comments under the review._
## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (kkmanuu) [reply index.html:333]: Done in displaying the button to the left.

**Comment by kkmanuu** [comment]
Making changes in displaying the button to the left.

**Review by DuaneDave — APPROVED** [review]
Hi @kkmanuu 👋 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✔️ Good job respecting the template

## Optional suggestions
N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code. Some of them were missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**